src/plugins/util/TooltipManager.js

```diff
@@ -0,0 +1,147 @@
+'use strict';
+
+/**
+ * TooltipManager
+ * Allows you to dynamically add tooltips to any HTML element.
+ * Usage: tooltipManager.setTooltip(element, 'Tooltip text', 'top');
+ */
+class TooltipManager {
+    constructor() {
+        this.tooltipElement = null;
+        this.createTooltipElement();
+    }
+
+    /**
+     * Creates the DOM element for the tooltip (unique, reused)
+     */
+    createTooltipElement() {
+        this.tooltipElement = document.createElement('div');
+        this.tooltipElement.className = 'gm-tooltip';
+        document.body.appendChild(this.tooltipElement);
+
+        const body = document.createElement('div');
+        body.className = 'gm-tooltip-body';
+        this.tooltipElement.appendChild(body);
+
+        // Ajoute la flèche
+        const arrowElement = document.createElement('div');
+        arrowElement.className = 'gm-tooltip-arrow';
+        this.tooltipElement.appendChild(arrowElement);
+    }
+
+    /**
+     * Adds a tooltip to an HTML element
+     * @param {HTMLElement} element - The target element
+     * @param {string} text - The tooltip text
+     * @param {string} [position] - 'top'|'bottom'|'left'|'right' (optional)
```

Comment:
For this position preference, maybe you could put the default order either in the doc, or as a default value in the function arguments?